### DISCUSSION TITLE
Issue Template consolidation in the organization vs. the repo

### DISCUSSION THREAD

**Discussion by kselena** [EddieHubCommunity/support#6401] [General]
The original issue can be found here: https://github.com/EddieHubCommunity/good-first-issue-finder/issues/81

However, it has grown into more of a general discussion as to whether or not all or some of the repos should utilize the organization's issue templates.  

### **Decisions to be made:** 
1. Org template finalization 
    >💡 Come to an agreement on what information we want in the org issue templates.
2. Repo level issue template set up
    >💡 Come to an agreement which repos will FULLY utilize org templates.  "Fully" meaning a repo either uses its own templates or the org templates.  The repo can't use both. 
3. Discussion about maximum issue template reusability for each repo (across the organization)
    >💡What this means is that some repos contain issue templates that are only unique to the repo itself and not the organization.  If we were to keep that unique issue template in the repo, it would prevent the repo from utilizing ANY of the organization level templates.  I think it would be best to either move the unique templates to another place OR find a way to see if those unique templates are _**REALLY**_ unique. They might be able to be utilized across the entire organization for the rest of the repos to use.
    > A good example of a unique level repo is the [Testimonal Template](https://github.com/EddieHubCommunity/LinkFree/blob/main/.github/ISSUE_TEMPLATE/testimonial.yml) in the **_LinkFree_** repository.  One way to think about this is that if this feature just opened its own page to create testimonials, then we would not need the issue template anymore.  These kinds of decisions could help drive the organization as a whole to take advantage of the reusability features of GitHub.

**Comment by eddiejaoude** [comment]
Thanks Kristen, I think we definitely want to have a standard across our repos in the Org - some repos may have extra templates in addition but they should all have the base templates
